src/gsi/intjo.f90

```diff
@@ -255,6 +258,7 @@ subroutine intjo_(rval,qpred,sval,sbias)
 class(obOper),pointer:: it_obOper
 
 !******************************************************************************
+  call setrad(sval(1))
```

Comment:
Why is this call added?

Reply:
Removed.

Reply:
Remove `use intradmod, only: setrad` since it is no longer needed.

Reply:
Done!

Reply:
Thank you.